python/pipeline/experiment.py

```diff
@@ -383,6 +384,16 @@ class Laser(dj.Part):
         gdd: float  # gdd setting
         """
 
+    def get_local_filename(self):
```

Comment:
get_local_tif_wildcard does not sound quite right to me , i put local_filename_with_tif_wildcard, it's very long but i couldn't think of something better